extra_geom/tests/test_generic_geometry.py

```diff
@@ -0,0 +1,85 @@
+from extra_geom import GenericGeometry
+
+import extra_geom
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+pixel_size = 0.01
+
+simple_config = {'pixel_size': pixel_size,
+                 'slow_pixels': 64,
+                 'fast_pixels': 128,
+                 'corner_coordinates': [pixel_size * np.array([5, 1, 0]),
+                                        pixel_size * np.array([-133, 5, 0]),
+                                        pixel_size * np.array([-63, -130, 0])],
+                 'n_tiles_per_module': 2,
+                 'ss_vec': np.array([1, 0, 0]),
+                 'fs_vec': np.array([0, 1, 0]),
+                 'tile_gap': 2 * pixel_size,
+                 'tile_vec': [0, -1, 0]
+                 }
+
+
+@pytest.fixture
+def simple():
+    return GenericGeometry.from_simple_description(**simple_config)
+
+
+def test_simple_geometry(simple):
+    """ Tests that an object was created """
+    assert isinstance(simple, extra_geom.detectors.GenericGeometry)
+    assert simple.expected_data_shape == (3, 64, 256)
+
+
+def test__tile_silce(simple):
+    _slice = simple._tile_slice(1)
+    assert isinstance(_slice, tuple)
+    assert isinstance(_slice[0], slice)
+    assert isinstance(_slice[1], slice)
+    assert _slice == (slice(0, 64), slice(128, 256))
+
+
+def test_write_crystfel_geom(simple):
+    simple.write_crystfel_geom('test.geom')
```

Comment:
```suggestion
def test_write_crystfel_geom(simple, tmp_path):
    simple.write_crystfel_geom(tmp_path / 'test.geom')
```

If a test writes a file, it's generally nice to do it in a temporary directory rather than just wherever you're running the tests.

Reply:
Agree, my bad. Thanks!